scipy/stats/tests/test_distributions.py

```diff
@@ -1412,6 +1412,11 @@ def get_moms(lam, sig, mu):
         sts = stats.exponnorm.stats(K, loc=mu, scale=sig, moments='mvsk')
         assert_almost_equal(sts, get_moms(lam, sig, mu))
 
+    def test_npnan_raises_error(self):
+        with pytest.raises(RuntimeError) as excinfo:
```

Comment:
you can use `assert_raises`, that is aligned to other tests

Maybe add another test for `np.inf`?

Reply:
added a test for `np.inf`